docs/conf.py

```diff
@@ -244,24 +244,15 @@ def _replace_missing_tokencredential_reference(
 
     ref_target = 'azure.core.credentials.TokenCredential'
 
-    try:
-        return reference(
-            ref_target,
-            ref_target,
-            internal=False,
-            refuri='https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/credentials.py',
-        )
-    except KeyError:
-        return None
+    return reference(
+        ref_target,
+        ref_target,
+        internal=False,
+        refuri='https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/credentials.py',
+    )
 
 
 def setup(app: Sphinx) -> dict[str, bool | str]:
-    """Register project-local Sphinx extension-API customizations.
-    :param app: Initialized Sphinx app instance.
```

Comment:
@jessicamack param descriptions might still be useful. It's the types that don't need to be duplicated.